akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpHeader.scala

```diff
@@ -26,6 +26,7 @@ abstract class HttpHeader extends jm.HttpHeader with ToStringRenderable {
   def lowercaseName: String
   def is(nameInLowerCase: String): Boolean = lowercaseName == nameInLowerCase
   def isNot(nameInLowerCase: String): Boolean = lowercaseName != nameInLowerCase
+  def safeToString: String = name
```

Comment:
I would've been OK with overriding `toString` as well, but this is :+1: to me too - perhaps it is good to be explicit about it.